sqoop/sqoop.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#    Copyright 2018 Google, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# This script installs Apache Sqoop (http://sqoop.apache.org/) on a Google Cloud
+# Dataproc cluster. Hive-Hcatalog is not installed automatically and if you want
+# to run hive jobs than please use this script with hive-hcatalog.sh init action.
+# HBase libraries are not installed automatically so in order to run imports to 
+# BigTable using sqoop, please run this init action with bigtable.sh.
+
+set -euxo pipefail
+
+readonly SQOOP_HOME='/usr/lib/sqoop'
+readonly SQOOP_CODE_LINK='https://github.com/apache/sqoop'
+readonly BIGTABLE_HBASE_CLIENT='bigtable-hbase-1.x-hadoop-1.3.0.jar'
```

Comment:
If you're requiring the BT init action, is this still necessary?

Reply:
After adding HBase libs to classpath in bigtable init actions it is not needed now.

Reply:
Right now when sqoop is not compiled on the cluster, but downloaded as compiled jar from somewhere else, we need to provide a minimal set of dependencies, because during compilation several jars were downloaded and added to classpath [see: bin/configure-sqoop](https://github.com/szewi/sqoop/blob/sqoop-bigtable-client/bin/configure-sqoop#L206). There are several already available on dataproc clusters, but the only missing is avro-1.8.1, so I downloaded it manually and added to init action script.

Reply:
`bigtable-hbase-1.x-hadoop-1.3.0.jar` connector is a different story. When I use BigTable init action along with sqoop, the BigTable init action already installs this jar file in `/usr/lib/hbase/lib` and put those on the classpath. Then it's being passed to sqoop in [bin/configure-sqoop](https://github.com/szewi/sqoop/blob/sqoop-bigtable-client/bin/configure-sqoop#L175), but it doesn't work with class not found exception.(I even printed sqoop classpath and bigtable-hbase jar is there!) 
Solution is either to symlink or copy `bigtable-hbase-1.x-hadoop-1.3.0.jar` to /usr/lib/sqoop/lib directory, which is a natural way of installing sqoop connectors. 
I was thinking about automatically detecting which init actions were used to create a cluster and then if there was a `bigtable` init action getting/linknig right jar file, but this doesn't seem to me like a proper way of handling that, explicitily installing connector will present to users how to easily add more connectors, what do you think @karth295 ?